.goreleaser.yml

```diff
@@ -15,3 +15,5 @@ builds:
       - linux
     goarch:
       - amd64
+    ldflags:
+      - -s -w -X version.Version={{.Version}} -X version.Commit={{.Commit}}
```

Comment:
You may want to add a comment here explaining why you add `-s` and `-w` with a link to https://golang.org/cmd/link/